src/open_r1/rewards.py

```diff
@@ -352,7 +352,12 @@ def evaluate_code(code, test_cases):
                 continue
 
             output = process.stdout.strip()
-            if output.strip() == case["output"].strip():
+
+            all_correct = True
```

Comment:
Let's add a comment to remind us in future why we did this:
```suggestion
            # TODO: implement a proper validator to compare against ground truth. For now we just check for exact strong match on each line of stdout.
            all_correct = True
```